src/config.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'bob'
```

Comment:
Let's use `.get('SECRET_KEY', 'bob')` here. Since `os.environ` is just a python dictionary we can use a second argument of `.get()` method to get some default value if key wasn't found in dictionary.

Yep, now `.get(...) or ...` works but it's not obvous why, and it's not the best solution here. And I'll try explain.

First of all we need to ask a question - what do we want to do by this line of code? And the answer is pretty simple - we want to get value of `SECRET_KEY` from environment variables dictionary and if it does not have a value for such key, then use default value `"bob"`.

Now let's try to figure out how `.get("SECRET_KEY") or "bob"` does this work?
`.get("somekey")` dictionary method tries to get value by provided key, but if dictionary has no such key, it returns `None`. So if there is no `"SECRET_KEY"` in our `os.environ` then `os.environ.get("SECRET_KEY")` will return `None`. After that we get expression like `None or "bob"`. And this expression returns `"bob"` because `or` operator always returns first non-falsy value. And this is `"bob"` in our case.

But let's image that someone provided environment variable `SECRET_KEY` with value `""` (empty string). Then our `os.environ` dictionary will actually _have_ a value for key `"SECRET_KEY"` and this value is an empty string. And now we expect that our `os.environ("SECRET_KEY") or "bob"` will return us a value from dictionary (an empty string `""`) but it doesn't. It still returns `"bob"`. Why? Because `.get("SECRET_KEY")` will return us and empty string and we will get an expression `"" or "bob"`. Since empty string is a falsy value, then `"" or "bob"` will return `"bob"` (`or` always returns first non-falsy value). And this behaviour doesn't meet our intentions - we wanted to get `"bob"` only if `os.environ` _doesn't have_ a value for `"SECRET_KEY"`.

Maybe here it will not be so important and such unexpected behaviour will never happen, because probably no one will set secret key to be an empty string. But this hidden trap will make us hurt in some other places where we will use `.get(...) or ...`.

So the rule is simple -  if you want to provide a default value for missing dictionary key never use `.get(...) or ...` . Always use `.get(..., ...)` instead.